static/css/perdiem.css

```diff
@@ -299,3 +299,22 @@ ul.tabs li {
     margin: 0 auto;
     padding: 20px;
 }
+
```

Comment:
I would create a new folder `extras/` under the `css/` directory and in there create a new file `resources.css` with these styles. Then in the template, you can load this new CSS file by adding:
```html
{% block extrastyle %}
    <link rel="stylesheet" href="{% static "css/extra/resources.css" %}" type="text/css" charset="utf-8" />
{% endblock %}
```

This block should go before the `content` block.